package.json

```diff
@@ -83,5 +83,9 @@
         "react-native-device-info": {
             "optional": true
         }
+    },
+    "publishConfig": {
+        "tag": "Release4.x"
```

Comment:
the release 4.x breach added tag under applicationinsights-react-native folder, but forgot to add tag under root package.json